src/charts/bar.js

```diff
@@ -105,6 +105,7 @@ define(function(require) {
             animationStepRatio = 70,
             interBarDelay = (d, i) => animationStepRatio * i,
 
+            highlightBarFunction = (bar) => d3Selection.select(bar).attr('fill', ({name}) => d3Color.color(colorMap(name)).darker()),
```

Comment:
Now seeing this, I am wondering if it will make sense to pass the selection of the bar to the function:
```
(barSelection) => barSelection.attr('fill', ...
```
This could make it a bit easier for the user to pass the new function? Or not?
What do you guys think?

Reply:
@Golodhros I think this totally makes sense. In this case, we are abstracting the usage of d3 from the user - which is I think what we aim for? I don't see other cases where the bar rectangle `<rect>` tag would be used by `d3` other than select. Unless there is any more possible ways do something with the passed `bar`. In short, I agree.